static-checks/ansible/syntax-check/test.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python3
+
+import subprocess
+
+from lib import ansible, util, oscap, results
+
+
+ansible.install_deps()
+
+ds = oscap.global_ds()
+all_profiles = ds.profiles.keys()
+
+# Generated playbooks
+for profile in all_profiles:
+    # Get ARF from scan
+    cmd = [
+        'oscap', 'xccdf', 'eval', '--profile', profile,
+        '--results-arf', 'arf.xml', ds.path,
+    ]
+    ret = util.subprocess_run(cmd, stdout=subprocess.DEVNULL)
```

Comment:
Why would you suppress the output here and for `generate fix` below? Is it too verbose (100s of KB or MBs)?

Because the output of ie. `oscap` is useful for figuring out whether it froze, and if it isn't excessively large, I would prefer to keep it in the test `output.txt` log.

Reply:
Sure, I did suppress it to not polish output because I see that output as not important there as it's about testing playbook syntax.

In `generate fix`, the output is minimal. Some empty lines or so. I'll remove the `stdout=subprocess.DEVNULL` as `generate fix` doesn't polish output much there.
But I do it also for `xccdf eval` few lines above. This one prints all rule results and even with `--progress` it will be quite verbose (scanning with every profile). What about this one? Are you fine with `stdout=subprocess.DEVNULL` there or would you rather see rule results there for figuring out if oscap frozen?

Reply:
`generate fix` doesn't output much, but `oscap xccdf eval` outputs all rule results and that's quite verbose even with `--progress` (scan for every profile). Are we fine with that verbose output which isn't specifically relevant for this test case where we test playbook syntax?

Reply:
I think I would rather have `output.txt` be the ultimate debug log (size within reason, we don't want >1GB logs), so I would say both would be fine. But I don't mind either way, I wast just curious.